### PR TITLE
[cert.pl] Add option to suppress stack traces.

### PR DESCRIPTION
**Pull request by ericwhitmansmith**
Disclaimer: I don't know perl.  Someone who does should look this over.

### CODE REVIEW

File: books/build/make_cert_help.pl [hunk 1]
```diff
@@ -580,7 +581,7 @@ sub parse_certify_flags
 $instrs .= "#+(and hons (not acl2-par)) (profile-fn 'certify-book-fn)\n";
 $instrs .= "(acl2::lp)\n";
 # We used to comment this out, but maybe it's better to leave this enabled by default?
-$instrs .= "(set-debugger-enable :bt)\n";
+$instrs .= "(set-debugger-enable :bt)\n" if ($STACK_TRACES);;
```
Comment (ragerdl) [inline books/build/make_cert_help.pl:584]: Is the double semi-colon here intentional?

**Comment by solswords** [comment]
Seems fine, other than the double-semi (which probably doesn't do anything?)

**Comment by ericwhitmansmith** [comment]
Double semi was not intentional, so I fixed it.   Thanks!

**Comment by MattKaufmann** [comment]
Eric, would you document the new NO_STACK_TRACES environment variable, perhaps somewhere under the :doc for cert.pl?

**Comment by ericwhitmansmith** [comment]
I'd be happy to do that, but I could use some guidance on where to put it.  I looked for a precedent, but I actually don't see any xdoc for most of the other environment vars in that section (e.g., ON_FAILURE_COMMAND, or ACL2_BOOKS_DEBUG, whatever those are).  Maybe I missed it.  Or maybe we can take documenting these vars off the critical path for accepting this change.

**Comment by MattKaufmann** [comment]
Good point, Eric; I didn't know (or remember) about the others.  Yes, I agree that documentation shouldn't be a requirement for accepting the pull request.

**Pull request merged**